policy-controller/k8s/index/src/outbound_index.rs

```diff
@@ -322,27 +328,71 @@ fn convert_backend(
     services: &HashMap<String, ServiceInfo>,
 ) -> Option<Backend> {
     backend.backend_ref.map(|backend| {
-        if let Some(port) = backend
+        if !is_backend_service(&backend.inner) {
+            return Backend::InvalidDst {
+                weight: backend.weight.unwrap_or(1).into(),
+                message: format!(
+                    "unsupported backend type {group} {kind}",
+                    group = backend.inner.group.as_deref().unwrap_or("core"),
+                    kind = backend.inner.kind.as_deref().unwrap_or("core"),
```

Comment:
default should be "Service" I think

Reply:
good catch -- my defaulting was off in the matcher below.

In this case, i've changed the value to `<empty>` -- in this case we know it's not a normal core Service; so we really don't know what the inferred kind should be if the group is specified.